src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderCharge.kt

```diff
@@ -129,6 +131,9 @@ class OffenderCharge(
   // always populated in prod but presumably won't be by DPS
   val lidsOffenceNumber: Int? = null,
 
+  @OneToMany(mappedBy = "offenderCharge", cascade = [CascadeType.REMOVE])
+  val offenderSentenceCharges: List<OffenderSentenceCharge> = mutableListOf(),
```

Comment:
remove parent OffenderSentenceCharges if OffenderCharge is deleted before a sentence is removed